cloudbuild-release.yaml

```diff
@@ -0,0 +1,14 @@
+# This cloudbuild is run on the creation of new tags, which should signify releases.
+steps:
+# Build the container that will do the go build.
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['build', '-t', 'builder', '-f', 'Dockerfile.build', '.']
```

Comment:
I don't see `Dockerfile.build` anywhere, do you need to check it in?

Reply:
It should have just been added in https://github.com/GoogleCloudPlatform/container-diff/commit/b726215b8b978e1d85257af5c2fd0a6fb8e116fd

Reply:
Ah yep looks like my pull didn't finish before I said that, spoke too soon :)